src/main/java/de/tum/in/www1/artemis/service/connectors/VersionControlService.java

```diff
@@ -143,18 +149,18 @@ enum RepositoryPermissions {
     /**
      * Remove the user from the repository
      *
-     * @param repositoryUrl     The repository url of the repository from which to remove the user. It contains the project key & the repository name.
-     * @param user              User which to remove from the repository
+     * @param repositoryUrl The repository url of the repository from which to remove the user. It contains the project key & the repository name.
+     * @param user          User which to remove from the repository
      */
     void removeMemberFromRepository(VcsRepositoryUrl repositoryUrl, User user);
 
     /**
      * Removes the user's write permissions for a repository.
      *
-     * @param repositoryUrl     The repository url of the repository to update. It contains the project key & the repository name.
-     * @param projectKey        The projectKey that the repo is part of in the VCS.
-     * @param users             Set of users for which to change permissions
-     * @throws VersionControlException        If the communication with the VCS fails.
+     * @param repositoryUrl The repository url of the repository to update. It contains the project key & the repository name.
+     * @param projectKey    The projectKey that the repo is part of in the VCS.
+     * @param users         Set of users for which to change permissions
+     * @throws VersionControlException If the communication with the VCS fails.
```

Comment:
You have many of these reformatting. Do we have a guideline for that?
I've also seen other PRs do the reformatting the other way round

Reply:
I am not entirely sure where this comes from. I use Gradle spotlessApply before committing. Some people might use Prettier?